pms_sale/models/sale_order.py

```diff
@@ -36,5 +36,5 @@ def action_confirm(self):
                 [("sale_order_id", "=", sale.id)]
             )
             if reservation:
-                reservation.action_book()
+                reservation.action_confirm()
```

Comment:
@JorgeJuarezCasai The reservation is confirmed when the invoice is paid:
https://github.com/ursais/pms/blob/14.0-ADD-pms_sale/pms_sale/wizards/account_payment_register.py